taxonomies/food/categories.txt

```diff
@@ -41476,6 +41481,11 @@ ciqual_food_code:en: 53503
 ciqual_food_name:en: Yam or Indian potato, peeled, boiled/cooked in water
 ciqual_food_name:fr: Igname, épluchée, bouillie/cuite à l'eau
 
+< en:Cereals and potatoes
```

Comment:
Taxonomy wise, Klöse can be potato, meat (Fleischklöße), Quark (Topfenknödel), wheat (Hefeknödel) based, and more. Based on https://de.wikipedia.org/w/index.php?title=Klo%C3%9F&oldid=248171204#Varianten, putting all of them under the potato/cereal parent category sounds a bit inaccurate